scripts/lint-routing.sh

```
@@ -736,6 +742,197 @@ if [ "$SELF_TEST_MODE" -eq 1 ]; then
+    fi
+    if validate_role_token "tech-lead:agent-push" >/dev/null 2>&1; then
+        :
+    else
+        _sc_fail "role-token: tech-lead:agent-push rejected (expected accept)"
+        return_early=1
+    fi
+    if validate_role_token "sme-brewing" >/dev/null 2>&1; then
+        :
+    else
+        _sc_fail "role-token: sme-brewing rejected (expected accept)"
+        return_early=1
```

Shellcheck (0.11.0)

[warning] return_early appears unused. Verify use (or export if used externally).

(SC2034)